bootstrap.sh

```diff
@@ -1 +1,21 @@
 #!/bin/bash
+
+kubectl apply -f .infrastructure/namespace.yml
+kubectl create namespace mysql
+
+# Apply MySQL StatefulSet and ConfigMap
+kubectl apply -f .infrastructure/mysql/mysql-secret.yml
```

Comment:
Ensure that the file path `.infrastructure/mysql/mysql-secret.yml` is correct and matches your directory structure.